cabby/rvs/templates.py

```diff
@@ -287,13 +288,22 @@ def add_features_to_template(template: Text, entity: geo_item.GeoEntity) -> Tupl
 
     if landmark.main_tag:
 
-      template = template.replace("?UP?"+landmark_type.upper(),
+      template_after = template.replace("?UP?"+landmark_type.upper(),
                                   landmark.main_tag.capitalize())
 
-      template = template.replace(landmark_type.upper(),
+
+
+      template_after_2 = template.replace(landmark_type.upper(),
                                   landmark.main_tag)
 
-      entities_tags.append(landmark.main_tag)
+      if landmark.landmark_type != "start_point":
+
+        if template_after == template and template_after_2 == template:
+          print("22222222222", landmark, template)
```

Comment:
Why is there a print statement here and why is it printing 2222...?

Reply:
removed